commands.go

```diff
@@ -35,8 +35,8 @@ func init() {
 			return &cmdInjectConnect.Command{UI: ui}, nil
 		},
 
-		"lifecycle-sidecar": func() (cli.Command, error) {
-			return &cmdLifecycleSidecar.Command{UI: ui}, nil
+		"consul-sidecar": func() (cli.Command, error) {
```

Comment:
Can we make `lifecycle-sidecar` an alias for `consul-sidecar`? If we're not adding any mandatory flags then it would be backwards compat I think?

Reply:
Oh interesting, yeah! Would that mean just having lifecycle-sidecar return the consul-sidecar command, i.e:
```
"lifecycle-sidecar": func() (cli.Command, error) {
			return &cmdConsulSidecar.Command{UI: ui}, nil
```

Hmm, this command is only called by the pod-mutating-webhook, right? So wouldn't nobody but our own code be calling consul-sidecar? If that's the case, I feel like we wouldn't need to create the alias since within the webhook handler we just change which command gets run for the sidecar. Its not exposed in any way through the helm chart.

There are also some inject-connect flags, like lifecycle-sidecar-cpu-request that have been renamed, so I was thinking the helm values would accept `global.lifecycleSidecarContainer.resources.requests.cpu` and `global.consulSidecarContainer.resources.requests.cpu` but both would point to the consul-sidecar-cpu-request flag in consul-k8s.